### ISSUE TITLE
Planet configuration troubleshooting

### ISSUE THREAD

**Issue by hiroTochigi** [open-learning-exchange/open-learning-exchange.github.io#2438]
### Problem
In Step 1, you can get the admin registration page after executing `vagrant up prod`.
Then, if you close the page without registering, you cannot get the registration page again.

I encountered this problem, so I executed `vagrant halt prod` and `vagrant up prod`. This method did not work.
Also, I executed `vagrant destroy prod` and `vagrant up prod`, but the method did not work either.
Therefore, I had to wipe out the Planet directory and started again.

### Steps to reproduce the problem

In Step 1
1. Execute `vagrant up prod`
1. Open Planet by a browser
1. Close the browser without your registration
1. Open Planet again

### Screenshots

N/A

### Proposed solution

Put the warning which tells that you should not close your browser before your registration.

**Comment by hiroTochigi** [comment]
P.S.
This is my guess, but if I hard refresh, I might be able to get the admin registration page without completely starting over.
Does anyone have ideas?
Thank you

**Comment by ArranHL** [comment]
Hay @hiroTochigi, well vagrant up prod will bring your virtual machine up so you can access your planet, from there you should be able to access your local directory at http://localhost:3100 or http://localhost:3100/eng/login 
I have however on occasion found that this can get stuck but the easiest fix is just to do:

 CTRL + SHIFT + R which should reload the page and bring up the sign in option. 

![image](https://user-images.githubusercontent.com/47016295/59557115-9639a900-8f8e-11e9-9352-749407aebf82.png)

Do note that vagrant destroy prod will **destroy** your planet completely so vagrant up prod can't work after it, in essence it is trying to start up a planet that isn't there. 

We just went full Alderaan on that thing and there's nothing left :) 

Did you rebuild your new machine after you used vagrant destroy prod? The Admins will have to validate  your new one if so, all you need to do is create a new one, change your GitHub name to match and post in the chat. 
Let me know if I missed anything or am off somewhere :)

**Comment by hiroTochigi** [comment]
@ArranHL, Thank you so much for the response to the issue. 
My issue arose when I closed the Planet before I registered it. 
In other words, my Planet, Community,  did not communicate with Nation yet. 
Therefore, there was not my account in Nation yet.

My problem was that I could not get the below page after closing it without registration.

![vi-become-admin](https://user-images.githubusercontent.com/44413241/59557879-c7c06d80-8faa-11e9-8778-9f4c0aa2437d.png)

After closing the above page without registration, I got only the below page

![vi-registration-accepted](https://user-images.githubusercontent.com/44413241/59557891-2ab20480-8fab-11e9-8dcc-b534121a89cc.png)

From this page, I could not register as administration.
From this point, `vagrant halt prod` then `vagrant up prod` did not work.
Also `vagrant destroy prod` then `vagrant up prod` did not take me the below page either.

![vi-become-admin](https://user-images.githubusercontent.com/44413241/59557879-c7c06d80-8faa-11e9-8778-9f4c0aa2437d.png)

Honestly, I did not remember that I tried CTRL + SHIFT + R for this issue.
However, I remembered that I got the registration page again after I deleted all source code.
Then, do the below code again.

```
git clone https://github.com/open-learning-exchange/planet.git
cd planet
vagrant up prod
```
Basically, what I said is that if the issue could be solved only by CTRL + SHIFT + R, I would like to add the technique for the troubleshooting section in vi-configurations-vagrant.md
If not, I would like to put a warning sign which tells not to close the registration page before completing it.

**Comment by ArranHL** [comment]
Hi @hiroTochigi I think I understand, you did not complete the registration process before the window was exited, in that case CTRL + SHIFT + R would not assist unfortunately, to the best of my knowledge the planet is configured to a single admin locally at this point. 

If you commenced but did not complete the Admin creation step, you did the right thing starting over from scratch. 

I suppose a little cautionary note saying, "please do dot exit page before completion" might help someone in the same situation :)

**Comment by hiroTochigi** [comment]
@ArranHL Thank you so much for your comment. I appreciate it.

**Comment by Liew211** [comment]
@hiroTochigi this is a good idea.  I'm pretty sure if you quit then it makes an automatic config, so I agree with the fix @ArranHL suggested.  Go ahead :)

**Comment by irisb1701** [comment]
Something similar happened to me too, but not during registration, it was during logging in. I agree with the @ArranHL's note. Maybe also add that destroying and rebuilding is less of a last resort, and more of a likely solution when having problems on this step.
